testing-daml-models/daml-scripts/2_first_script.md

```diff
@@ -0,0 +1,45 @@
+Next to a fixed `operator` party we want to allocate a list of parties given as an argument to the
+`initialize` script. We want them to follow the operator and the operator should follow them back.
+As before, we will be using the `allocatePartyWithHint` function. Replace the initialize script
+with:
+
+<pre class="file" data-target="clipboard">
+initialize: [Text] -> Script ()
+initialize initialParties = do
+    operator <- allocatePartyWithHint "operator" (PartyIdHint "operator")
+    ps <- forA initialParties $ \partyStr -> do
+      allocatePartyWithHint partyStr (PartyIdHint partyStr)
+    debug "done!"
+</pre>
+
+Easy! Scripts are just like usual DAML functions and follow the same syntax to declare arguments and
+type signatures.
+
+1. Now we get the other party names from the `initialParties` argument to the script. 
+1. `forA: (Applicative m) => [a] -> (a -> m b) -> m [b]` starts a `for loop` that you can
+use to map anything that returns an `Applicative` (such as `Script`) over an input list and returns
```

Comment:
Do we need to mention here `Applicative` or is it OK if we omit it? It's another term the user needs to understand.

Reply:
I don't want to change the type signature of `forA`, since that's the way it's also in the docs.